website/docs/pipelines/promoting-applications.mdx

```diff
@@ -331,11 +426,87 @@ The value of the `$promotion` field in the comment is comprised of 3 components
 
 Weave GitOps Enterprise will look for this marker whenever it receives an event from the respective HelmRelease of one of the leaf clusters and patch the file with the version denoted in the event (see the section above for instructions on setting up notification events from leaf clusters). Finally, it will create a Git provider PR to update the version of the application for the next environment in the pipeline.
 
+### Supported Git Providers
```

Comment:
I think from here down it's just moved text, is that right? (feel free to ignore comments below here!)

Reply:
Umms, I could not see comments for this section: feel free to add them to review them too. Thanks!

Reply:
There was just one I think; I didn't bother reviewing most of the text that you didn't change.